index.js

```diff
@@ -68,7 +68,7 @@ function PinoColada () {
     if (!obj.name) obj.name = ''
     if (!obj.ns) obj.ns = ''
 
-    output.push(formatDate())
+    output.push(formatDate(obj.time || Date.now()))
```

Comment:
actually going to use `new Date()` instead of Date.now() as otherwise this would be a breaking change.
```suggestion
    output.push(formatDate(obj.time || new Date()))
```